ktoml-core/src/commonTest/kotlin/com/akuleshov7/ktoml/decoders/custom/CustomSerializerTest.kt

```diff
@@ -0,0 +1,335 @@
+package com.akuleshov7.ktoml.decoders.custom
+
+import com.akuleshov7.ktoml.Toml
+import kotlinx.datetime.LocalDate
+import kotlinx.serialization.KSerializer
+import kotlinx.serialization.Serializable
+import kotlinx.serialization.builtins.IntArraySerializer
+import kotlinx.serialization.decodeFromString
+import kotlinx.serialization.descriptors.*
+import kotlinx.serialization.encoding.CompositeDecoder
+import kotlinx.serialization.encoding.Decoder
+import kotlinx.serialization.encoding.Encoder
+import kotlinx.serialization.encoding.decodeStructure
+import kotlin.test.Test
+import kotlin.test.assertEquals
+
+class CustomSerializerTest {
```

Comment:
I remember, I had `CustomSerializerTest` with some @Ignored tests

kotlin/com/akuleshov7/ktoml/decoders/CustomSerializerTest.kt

May be unify and remove @Ignored in some of tests? 🙏🏾

Reply:
It's the same class, I just moved it to */custom/* directory